src/CMakeLists.txt

```diff
@@ -172,7 +172,7 @@ if(DLAF_WITH_GPU)
   )
   target_sources(dlaf.core PRIVATE ${dlaf_core_gpu_source_files})
   if(DLAF_WITH_HIP)
-    set_source_files_properties(${dlaf_core_gpu_source_files} PROPERTIES LANGUAGE CXX)
+    set_source_files_properties(${dlaf_core_gpu_source_files} PROPERTIES LANGUAGE HIP)
```

Comment:
It will be probably useful to have a CMake helper function to set properties for gpu source files, just to easily unify their management.

I will open an issue to keep track of the idea.